yaml/pois.yaml

```diff
@@ -260,6 +260,9 @@ filters:
   - filter: {tourism: winery}
     min_zoom: GREATEST(14, LEAST(zoom + 2.85, 17))
     output: {kind: winery}
+  - filter: {tourism: gallery}
+    min_zoom: GREATEST(14, LEAST(zoom, 17))
```

Comment:
@nvkelso can you verify that it's ok for huge art galleries to start appearing at z14?